experimenter/manifesttool/tests/test_cli.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+from unittest import TestCase
+from unittest.mock import patch
+
+from click.testing import CliRunner
+import yaml
+
+from manifesttool import cli
+from manifesttool.appconfig import AppConfig, AppConfigs
+
+
+# An AppConfig with a blank repo and fml_path so that the CLI cannot fetch
+# anything.
```

Comment:
Ah okay so this tests that we can parse the app config file, trigger the cli in a networkless no op mode, and then write out and exit successfully.  Cool.

Reply:
Where does this test get invoked?

Reply:
It gets invoked by pytest

Reply:
Oh pytest just discovers and runs it.  Well so much for my plan to deprecate it.  Cool 👍

Reply:
You can see it in the test run [here](https://app.circleci.com/pipelines/github/mozilla/experimenter/29340/workflows/d678a55e-11e2-4e94-8d7a-31f2263b9679/jobs/112000/parallel-runs/0/steps/0-104). My next push will include it in coverage, but my local test run shows 100% coverage:
```
manifesttool/tests/test_cli.py::FetchLatestTests::test_valid_manifest PASSED                                                                                                                              [100%]

---------- coverage: platform linux, python 3.11.1-final-0 -----------
Name                                                                       Stmts   Miss  Cover   Missing
--------------------------------------------------------------------------------------------------------
manifesttool/__init__.py                                                       0      0   100%
manifesttool/appconfig.py                                                     18      0   100%
manifesttool/cli.py                                                           26      0   100%
manifesttool/nimbus_cli.py                                                    10      0   100%
```